sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPSensor.java

```diff
@@ -188,10 +188,9 @@ private void analyseFile(SensorContext context, PHPAnalyzer phpAnalyzer, InputFi
 
     } catch (RecognitionException e) {
       checkInterrupted(e);
-      LOG.error("Unable to parse file: " + inputFile.filename());
+      LOG.error("Unable to parse file [{}] at line {}", inputFile.uri(), e.getLine());
```

Comment:
why not use `'` instead of `[ ]`? 

`"Unable to parse file '{}' at line {}"` ?

Reply:
it's debatable of course, but I prefer more visible separators because URI of file might be pretty long